### PR TITLE
Redo RubinTV frontend

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

**Comment by jonathansick** [comment]
It looks like the CI errors aren't related to having secrets or not. Unless you want help on pin-pointing the issues, feel free to ping me when you're ready for the review and I'll take care of it.

**Comment by mfisherlevine** [comment]
> It looks like the CI errors aren't related to having secrets or not. Unless you want help on pin-pointing the issues, feel free to ping me when you're ready for the review and I'll take care of it.

As discussed OOB, I think these are all sorted now, and things are green so should be good to go 🤞

File: src/rubintv/templates/cameras/historical.jinja [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        </div> {# end of year #}
+      {% endfor %}
+  </section>
+  <section class="channel-day-data">
+    {% include "cameras/day-data-per-day-channels.jinja" %}
+  </section>
+</div>
+{% endblock content %}
+{% block footer_scripts %}
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
```
Comment (jonathansick) [inline src/rubintv/templates/cameras/historical.jinja:37]:
```suggestion
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
```

**Review by jonathansick — APPROVED** [review]
Looks good. I've got some comments about organization and enhancing the type checking, but you can get to those later if you want.

It might be good to have the sass code outside the Python app and then up a build pipeline (anything from Make to gulp and beyond) build those artifacts in CI. That way you don't need to keep the css in version control and have things come out of sync. Up to you.

**Pull request merged**